incubator/hnc/internal/kubectl/tree.go

```diff
@@ -81,6 +81,21 @@ func printSubtree(prefix string, hier *api.HierarchyConfiguration, inCycle bool)
 		if cycle && inCycle {
 			continue
 		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		ns, err := k8sClient.CoreV1().Namespaces().Get(ctx, txt, metav1.GetOptions{})
+		if err != nil {
+			fmt.Printf("Could not get namespaces: %s\n", err)
+			os.Exit(1)
+		}
+
+		_, hasSubnamespaceAnnotation := ns.ObjectMeta.Annotations[api.SubnamespaceOf]
+		if hasSubnamespaceAnnotation {
+			txt = "[s]" + txt
```

Comment:
Need a space after the square bracket

`[s] foo`

Not

`[s]foo`

Reply:
done